acis_thermal_check/main.py

```diff
@@ -1067,10 +1072,10 @@ def make_validation_plots(self, tlm, model_spec, outdir, run_start):
         f.write(quant_table)
         f.close()
 
-        # If run_start is specified this is likely for regression testing
+        # If self.write_pickle is specified this is likely for regression testing
```

Comment:
Actually "specified" is a much better comment.

Comments are not supposed to replicate/repeat the code: the reader can see how the if statement works. Comments are there to tell the reader  what the intent is....to explain what's going on so that the reader does not have to waste time figuring out what's going on.

As a simple example, suppose you are incrementing x:

x +=1

The following is a useless comment:

**# Increment x**
x +=1

Anyone can see that x is being incremented. The comment provides no additional information.
The reader does not know why you are doing that.

But this is a more useful comment:

**# Point to the next item in the  star chart list.**
x+=1

"specified" is better than True. But it can be made better still. I changed it to:

        # self.write_pickle is set to the value of the --run-start command 
        # line argument; either a DOY date string or None (if the argument 
        # was not specified). If a DOY date, this model run is likely for regression 
        # testing or other debugging.  In that case write out the full 
        # predicted and telemetered dataset as a pickle.

Anyone trying to figure out what we did now knows how self.write_pickle gets set, why it is set
and the intent of the block of code.